src/main/java/org/junitpioneer/jupiter/IssueExtension.java

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright 2015-2020 the original author or authors.
+ *
+ * All rights reserved. This program and the accompanying materials are
+ * made available under the terms of the Eclipse Public License v2.0 which
+ * accompanies this distribution and is available at
+ *
+ * http://www.eclipse.org/legal/epl-v20.html
+ */
+
+package org.junitpioneer.jupiter;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.extension.BeforeEachCallback;
+import org.junit.jupiter.api.extension.ExtensionConfigurationException;
+import org.junit.jupiter.api.extension.ExtensionContext;
+import org.junit.jupiter.api.extension.ExtensionContext.Namespace;
+import org.junit.platform.commons.support.AnnotationSupport;
+
+/**
+ * This class implements the functionality for the {@code @Issue} annotation.
+ *
+ * @see Issue
+ */
+@SuppressWarnings("ALL")
```

Comment:
I'm sorry but: why?

Reply:
Wrong usage. Wanted to supress only this `unchecked` call (as I don't understand how to implement it correctly to avoid the compiler warning ... ). Changed.